src/debugger/mock-ui/UISession.js

```diff
@@ -280,6 +289,17 @@ export class UISession {
         };
         this._sendStepIntoRequest(stepIntoArgs);
         break;
+      case "eval":
+        if (parts.length < 3) return false;
+        let evalFrameId = parseInt(parts[1], 10);
+        if (isNaN(evalFrameId)) return false;
```

Comment:
This forces user to specify evalFrameID. Do you have a CLI command to perform global eval(without specify frameId?)